### ISSUE TITLE
Can you release the details of training GooLeNet?

### ISSUE THREAD

**Issue by liangbh6** [pkuvmc/PTGAN#2]
Hi, I can't reproduce your results in Figure 8. I mean the baseline such as training on Market and directly testing on Duke. I want to know the network architecture, the optimizer, the learning rate, input size, embedding size, and so on. Hope that someone can help me.
